packages/checkbox/theme/lumo/vaadin-checkbox-styles.js

```diff
@@ -26,6 +26,13 @@ registerStyles(
       --_focus-ring-color: var(--vaadin-focus-ring-color, var(--lumo-primary-color-50pct));
       --_focus-ring-width: var(--vaadin-focus-ring-width, 2px);
       --_selection-color: var(--vaadin-selection-color, var(--lumo-primary-color));
+      --_helper-spacing: var(--vaadin-input-field-helper-spacing, 0.4em);
```

Comment:
I feel that `0.4em` space between the helper and the label (which already has a significant amount of white space below it) looks strange. I know it's the same in `text-field` etc, but those have the field chrome around the text so it's visually quite different.

Haven't tested it but I'd expect that space to be even more problematic in a vertical `checkbox-group`.

0.2em looks much more appropriate between label and helper.

Reply:
Changed to `0.2em` for both helper and error message.